README.md

```diff
@@ -8,8 +8,10 @@ Provides Azure Authentication Middleware for a Laravel App.  If you like this, c
 2. run `php artisan vendor:publish --provider="RootInc\LaravelAzureMiddleware\AzureServiceProvider"` to install config file to `config/azure.php`
 3. In our routes folder (most likely `web.php`), add
 ```php
-Route::get('/login/azure', '\RootInc\LaravelAzureMiddleware\Azure@azure');
-Route::get('/login/azurecallback', '\RootInc\LaravelAzureMiddleware\Azure@azurecallback');
+Route::get('/login/azure', '\RootInc\LaravelAzureMiddleware\Azure@azure')
```

Comment:
Let's add a step (3.a, 3.b?, or just a new int) that mentioned the possibility of naming routes.  If we name the route, we don't have to configure the callback on the portal.  If we don't, we need to make sure we add the URL in the portal.